services/notifications/README.md

```diff
@@ -35,6 +43,7 @@ templates
 ```
 
 Custom email templates referenced via `NOTIFICATIONS_EMAIL_TEMPLATE_PATH` must also be located in subfolders `shares` and `spaces` and must have the same names as the embedded templates. It is important that the names of these files and  folders match the embedded ones.
```

Comment:
This is no longer correct. We now only have one email template for all emails?

Reply:
fixed